packages/editor-ui/src/components/layouts/ResourcesListLayout.vue

```diff
@@ -301,12 +299,12 @@ export default mixins(
 			return this.$locale.baseText(`${this.resourceKey as IResourceKeyType}.menu.${this.isOwnerSubview ? 'my' : 'all'}`);
 		},
 		sendSubviewTelemetry() {
-			this.$telemetry.track('User changed credentials sub view', {
+			this.$telemetry.track(`User changed ${this.resourceKey} sub view`, {
 				sub_view: this.getTelemetrySubview(),
 			});
 		},
 		sendSortingTelemetry() {
-			this.$telemetry.track('User changed sorting in cred list', {
+			this.$telemetry.track(`User changed sorting in ${this.resourceKey} list`, {
```

Comment:
reminder to update telemetry table with new event and let david know....

Reply:
Done. David is aware and agreed to the change.